src/client/dfs/dfs.c

```diff
@@ -38,10 +38,23 @@
 #define LAYOUT_NAME	"DFS_LAYOUT_VERSION"
 /** A-key name of Default chunk size */
 #define CS_NAME		"DFS_CHUNK_SIZE"
-/** A-key name of Default Object Class */
-#define OC_NAME		"DFS_OBJ_CLASS"
-/** Consistency mode of the DFS container. */
+/** A-key name of Default Object Class for files */
+#define FILE_OC_NAME	"DFS_OBJ_CLASS"
+/** A-key name of Default Object Class for directories */
+#define DIR_OC_NAME	"DFS_DIR_OBJ_CLASS"
+/** Consistency mode of the DFS container */
 #define MODE_NAME	"DFS_MODE"
+/** A-key name of the object class hints */
+#define HINT_NAME	"DFS_HINTS"
+
+#define MAGIC_IDX	0
+#define SB_VER_IDX	1
+#define LO_VER_IDX	2
+#define CS_IDX		3
+#define FILE_OC_IDX	4
+#define DIR_OC_IDX	5
+#define CONT_MODE_IDX	6
+#define HINTS_IDX	7
```

Comment:
Suggestion - It would be more intuitive if all `*_NAME` and `*_IDX` followed the same pattern. For example, for:
```
#define MODE_NAME "DFS_MODE"
...
#define CONT_MODE_IDX 6
```
Instead:
```
#define CONT_MODE_NAME "DFS_MODE"
...
#define CONT_MODE_IDX 6
```
Where only `_NAME` and `_IDX` are different

Reply:
sure, ill make the change when i repush